examples/analysis_options.yaml

```diff
@@ -6,8 +6,8 @@ analyzer:
     implicit-casts: false
     implicit-dynamic: false
 
-  plugins:
-    - dart_code_metrics
+    #plugins:
+    #  - dart_code_metrics
```

Comment:
why??? 😢

Reply:
This plugin is messing with my LSP plugin, which was making it impossible to work.

I commented it to be able to work, but forgot the put it back.